index.html

```diff
@@ -469,13 +493,220 @@ <h4>Attributes</h4>
-              <var>track</var>.{{MediaStreamTrack/[[VideoStats]]}}.</p>
+              <p>Return <var>track</var>.{{MediaStreamTrack/[[Stats]]}}.</p>
             </li>
           </ol>
         </dd>
       </dl>
     </section>
+  </section>
+    <section>
+      <h4>The MediaStreamTrackAudioStats interface</h4>
+      <div>
+        <pre class="idl" data-cite="HR-TIME">[Exposed=Window]
+interface MediaStreamTrackAudioStats {
+  readonly attribute unsigned long long deliveredFrames;
+  readonly attribute DOMHighResTimeStamp deliveredFramesDuration;
+  readonly attribute DOMHighResTimeStamp deliveredFramesLatency;
+  readonly attribute DOMHighResTimeStamp? latency;
+  readonly attribute unsigned long long totalFrames;
+  readonly attribute DOMHighResTimeStamp totalFramesDuration;
+  [Default] object toJSON();
+};
+        </pre>
+        <div>
+          <p>The {{MediaStreamTrackAudioStats}} expose frame counters for the
+          {{MediaStreamTrack}} that created it. For this track, the user agent
+          is required to count each audio frame from its source as follows:</p>
+          <ul>
+            <li>
+              <p>A frame is considered a <dfn data-lt="delivered audio frames">
+              delivered audio frame</dfn> if it either was delivered to a sink
+              or would have been delivered to a sink, if one was connected.</p>
+            </li>
+            <li>
+              <p>The <dfn data-lt="delivered audio frames duration">delivered
+              audio frames duration</dfn> is the total duration of all
+              [= delivered audio frames =]. This measurement is incremented at
+              the same time as [= delivered audio frames =] and is measured in
+              milliseconds.</p>
+            </li>
+            <li>
+              <p>The <dfn data-lt="delivered audio frames latency">delivered
+              audio frames latency</dfn> is the sum of [= delivered audio
+              frames =]'s input latency. The input latency is the estimated
+              latency of the capture device plus the time it takes the user
+              agent to deliver that frame. This measurement is incremented at
+              the same time as [= delivered audio frames =] and is measured in
+              milliseconds. Note that once a frame has been delivered to a sink,
+              that sink may internally have its own latency (such as playout
+              delay) which is not included in this measurement.</p>
+            </li>
+            <div class="note">
+              <p>The average latency per audio frame can be calculated as
+              [= delivered audio frames latency =] / [= delivered audio
+              frames =].</p>
+            </div>
+            <li>
+              <p>The <dfn data-lt="current audio latency">current audio
+              latency</dfn> is the input latency of the last audio frame that
+              was delivered. If no audio frames have been delivered yet, this
```

Comment:
what's the unit?

Reply:
Seconds. But this should be in #119, sorry.